tribits/doc/guides/maintainers_guide/TribitsMaintainersGuide.rst

```diff
@@ -71,6 +71,8 @@ Developers`_ and `TriBITS System Architects`_.
 Appendix
 ========
 
+.. include:: ../TribitsCMakeLangaugeOverviewAndGotchas.rst
```

Comment:
`Language` has been misspelled here (again, maybe Codespell would catch this?)

Reply:
Will spell checking tools check the spelling of `CamelCase` names?  I think some doe if you use `lower_case` with words separated by '_'.

Reply:
Well, AFAICT Codespell does not split `CamelCase` names into their respective words, but there may be other tools that do (or perhaps someone could add such support to Codespell?)

Reply:
Addressed in commit 16b18171